### PR TITLE
Add lightbox to gallery

### PR DESCRIPTION
**Pull request by IvanaKos**

### CODE REVIEW

**Comment by IvanaKos** [comment]
I think this is all I can do with the free license.
There is also a VIEW MORE button at the bottom of the section. Does it make sense in combination with the lightbox? Add some functionality to it or remove it completely?

File: index.html [hunk 1]
```diff
@@ -113,9 +120,16 @@ <h2>Lorem ipsum dolor sit</h2>
                     <h2>Lorem ipsum dolor sit</h2>
                     <p>amet consetetur sadipscing elitr</p>
                   </div>
-                  <button class="btn-zoom">
-                    <img src="assets/img/magnifying-glass.png" alt="" />
-                  </button>
+                  <a
+                    data-fslightbox="lightbox-1"
+                    href="	https://ivanakos.github.io/IvanaKos.github.io-Variety/assets/img/gallery-img-2.jpg"
```
Comment (DekiGk) [inline index.html:125]:
```suggestion
                    href="https://ivanakos.github.io/IvanaKos.github.io-Variety/assets/img/gallery-img-2.jpg"
```

**Review by DekiGk — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -101,9 +101,16 @@ <h3>Finalize</h3>
                     <h2>Lorem ipsum dolor sit</h2>
                     <p>amet consetetur sadipscing elitr</p>
                   </div>
-                  <button class="btn-zoom">
-                    <img src="assets/img/magnifying-glass.png" alt="" />
-                  </button>
+                  <a
+                    data-fslightbox="lightbox-1"
+                    href="https://ivanakos.github.io/IvanaKos.github.io-Variety/assets/img/gallery-img-1.jpg"
```
Comment (DekiGk) [inline index.html:106]: As discussed, let's try with relative links

```suggestion
                    href="assets/img/gallery-img-1.jpg"
```

**Comment by DekiGk** [comment]
> I think this is all I can do with the free license. There is also a VIEW MORE button at the bottom of the section. Does it make sense in combination with the lightbox? Add some functionality to it or remove it completely?

Remove it completely.

File: index.html (hunk 1 above)
Comment (IvanaKos) [reply index.html]: Not needed since I changed the path to relative.

File: index.html (hunk 2 above)
Comment (IvanaKos) [reply index.html]: Done (not working locally now).

**Review by DekiGk — APPROVED** [review]

**Pull request merged**